tests/test_vlan_private.rb

```diff
@@ -387,7 +387,7 @@ def test_private_vlan_association_failure
 
       pv_type = 'isolated'
       assert_raises(RuntimeError, 'vlan misconf did not raise RuntimeError') do
-        v2.private_vlan_type = pv_type
+        v3.private_vlan_type = pv_type
```

Comment:
You have to look at the prev commands. v3 is vlan 200 but vlan 200 became part of an association, so now you can't isolate it, hence the RuntimeError.